stages/org.osbuild.ostree.coreos.aleph

```diff
@@ -56,11 +56,17 @@ SCHEMA_2 = """
 """
 
 def write_to_file(tree, filename, data):
-    with open(tree + filename, "w") as f:
+    with open(tree + "/" + filename, "w") as f:
         json.dump(data, f, indent=4, sort_keys=True)
         f.write("\n")
     f.close()
 
+    # create a symlink for backwards compatibility with CoreOS
```

Comment:
let's make the creation of the symlink an option for this stage and default it to false?

theoretically non coreos variants wouldn't want the backwards compatibility.